src/odbc/src/statement.cpp

```diff
@@ -47,9 +47,9 @@ Statement::Statement(Connection& parent)
     : connection(parent),
       columnBindings(),
       currentQuery(),
-      rowsFetched(0),
-      rowStatuses(0),
-      columnBindOffset(0),
+      rowsFetched(nullptr),
+      rowStatuses(nullptr),
+      columnBindOffset(nullptr),
```

Comment:
Need to add rowsetSize to the initializer list.

Reply:
Added. I have followed the implementation for initializing `rowArraySize` to `1`, though I'm not entirely sure why that is done.